autopts/ptsprojects/mynewt/gatt.py

```diff
@@ -180,7 +180,15 @@ def set_addr(addr):
 
     custom_test_cases = [
         ZTestCase("GATT", "GATT/SR/GAN/BV-02-C",
-                  pre_conditions_2,
+                  pre_conditions_2
+                        + [
+                            TestFunc(get_stack().synch.add_synch_element,
+                                     [SynchPoint("GATT/SR/GAN/BV-02-C", 1),
+                                      SynchPoint("GATT/SR/GAN/BV-02-C_LT2", 1)]),
+                            TestFunc(get_stack().synch.add_synch_element,
+                                     [SynchPoint("GATT/SR/GAN/BV-02-C_LT2", 308),
+                                      SynchPoint("GATT/SR/GAN/BV-02-C", 93)]),
+                      ],
```

Comment:
```suggestion
                  cmds=pre_conditions_2 + 
                  [TestFunc(get_stack().synch.add_synch_element,
                            [SynchPoint("GATT/SR/GAN/BV-02-C", 1),
                             SynchPoint("GATT/SR/GAN/BV-02-C_LT2", 1)]),
                   TestFunc(get_stack().synch.add_synch_element,
                            [SynchPoint("GATT/SR/GAN/BV-02-C_LT2", 308),
                             SynchPoint("GATT/SR/GAN/BV-02-C", 93)])],
```

Suggest to use same formatting as below

Reply:
Good suggestion - formatting is now somewhat more readable :)